### PR TITLE
fix: Fix FileManager logs warning for .DS_Store

### PR DESCRIPTION
**Pull request by philipphofmann**
## :scroll: Description

Don't log a warning in SentryFileManager for .DS_Store files.

## :bulb: Motivation and Context

Fixes one part of GH-3577

## :green_heart: How did you test it?
Unit tests

## :pencil: Checklist

You have to check all boxes before merging:

- [x] I reviewed the submitted code.
- [x] I added tests to verify the changes.
- [x] No new PII added or SDK only sends newly added PII if `sendDefaultPII` is enabled.
- [x] I updated the docs if needed.
- [x] Review from the native team if needed.
- [x] No breaking change or entry added to the changelog.
- [x] No breaking change for hybrid SDKs or communicated to hybrid SDKs.

### CODE REVIEW

**Comment by philipphofmann** [comment]
Still need to fix the tests, sorry.

File: Sources/Sentry/include/SentryFileManager.h [hunk 1]
```diff
@@ -55,6 +56,11 @@ SENTRY_NO_INIT
 
 - (void)deleteOldEnvelopeItems;
 
+/**
+ * Only used for testing.
```
Comment (armcknight) [inline Sources/Sentry/include/SentryFileManager.h:60]: Could we create a `SentryFileManager+Test.h` in the test target to expose this?

**Review by armcknight — COMMENTED** [review]
I'll wait to review until the tests pass, but there was one small detail I figured I'd comment on.

File: Sources/Sentry/include/SentryFileManager.h (hunk 1 above)
Comment (philipphofmann) [reply Sources/Sentry/include/SentryFileManager.h:60]: The FileManager isn't public, so I often don't add the extra header file. It's just extra work. Still, if you'd like, we could add one.

**Review by brustolin — APPROVED** [review]
LGTM

File: Sources/Sentry/include/SentryFileManager.h (hunk 1 above)
Comment (philipphofmann) [reply Sources/Sentry/include/SentryFileManager.h:59]: @armcknight, I'm going to merge this PR and maybe open another PR, depending on your reply.

**Pull request merged**